charts/vector/ci/vector-templating-values.yaml

```diff
@@ -17,7 +17,7 @@ customConfig:
       inputs: [generator]
       bucket: logs-archive
       key_prefix: >-
-        {{ print "{{kubernetes.pod_labels.app\\.kubernetes\\.io/client-id}}/%Y/%m/%d/{{kubernetes.pod_name}}/" }}
```

Comment:
This seems to have been making it through to Vector as `\\.`s.

Reply:
:D You mean we should follow the docs I recently updated 🙃 

https://vector.dev/docs/reference/configuration/template-syntax/

Reply:
Yes indeed 😄 I made the quoting changes in [a82cde9](https://github.com/vectordotdev/helm-charts/pull/275/commits/a82cde9f77b96727ba6a4f07078c27f1fad979ed)